demos/rag_agentic/notebooks/Level0_getting_started_with_Llama_Stack.ipynb

```diff
@@ -181,19 +118,12 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Connected to Llama Stack server @ http://localhost:8321\n"
+      "Connected to Llama Stack server\n"
      ]
     }
    ],
    "source": [
-    "remote = os.getenv(\"REMOTE\", \"True\")\n",
-    "\n",
-    "if remote == \"False\":\n",
-    "    local_port = os.getenv(\"LOCAL_SERVER_PORT\", 8321)\n",
-    "    base_url = f\"http://localhost:{local_port}\"\n",
-    "else: # any value non equal to 'False' will be considered as 'True'\n",
-    "    base_url = os.getenv(\"REMOTE_BASE_URL\")\n",
-    "\n",
+    "base_url = os.getenv(\"REMOTE_BASE_URL\", \"http://localhost:8321\")\n",
     "\n",
     "# Tavily search API key is required for some of our demos and must be provided to the client upon initialization.\n",
```

Comment:
Shall we remove this line since it appears to be already mentioned in the note above?

Reply:
I think its ok to leave it as is.